src/sentry/api/endpoints/organization_events_facets.py

```diff
@@ -55,4 +57,16 @@ def get(self, request: Request, organization) -> Response:
 
                 resp["project"]["topValues"] = filtered_values
 
+            if "device.class" in resp:
+                # Map device.class tag values to low, medium, or high
+                filtered_values = []
+                for v in resp["device.class"]["topValues"]:
+                    for key, value in DEVICE_CLASS.items():
+                        if v["value"] in value:
+                            name = key
+                            v.update({"name": name})
+                            filtered_values.append(v)
```

Comment:
we can probs add a `continue` at the end of this if block

Reply:
good call